apiserver/apiserver.go

```diff
@@ -93,6 +93,12 @@ func (api *API) Register(container *restful.Container) {
 		To(api.Status).
 		Doc("Returns 200 when the container is up"))
 
+	// Return information about the version of API server running
+	ws.Route(ws.
+		GET("/version").
+		To(api.Version).
+		Doc("Returns version information"))
+
```

Comment:
I wasn't exactly sure what we *should* be returning.  I looked at what Kubernetes returns and I liked at the _istioctl_ client side.  I will go with what our client side defines.